src/org/zaproxy/zap/extension/exportreport/filechooser/FileType.java

```diff
@@ -65,4 +67,8 @@ public String getDescription() {
     public ImageIcon getIcon() {
         return icon;
     }
+
+    public boolean getEnabled() {
```

Comment:
@thc202 changed but it's only a temporary addition until i finish and add in the other export types :D